bigtop-deploy/puppet/modules/gpdb/manifests/init.pp

```diff
@@ -137,6 +140,16 @@
             ensure   => latest,
           }
           package { ["paramiko"]:
+            ensure   => latest,
+            provider => pip,
+            require  => Package["setuptools"],
+          }
+          package { ["setuptools"]:
```

Comment:
Totally ignorant about the environment but are those packages named python-setuptools or simply setuptools?